tests/redirects/map_globalconf.py

```diff
@@ -1139,4 +1124,11 @@
 
     # Bug 1335040
     url_test('/gigabit/{,apply/}', 'https://learning.mozilla.org/gigabit/'),
+
+    # Bug 1329931
+    url_test('/styleguide/products/firefox-os/privacy/matters',
```

Comment:
Nit: it would be good to test a couple of different styleguide URLs here, like you have done for `/firefox/os` below